config/deploy.rb

```diff
@@ -13,7 +13,7 @@
 # Default value for linked_dirs is []
 set :linked_dirs, %w{log tmp/pids tmp/cache tmp/sockets vendor/bundle}
 
-set :branch, 'master'
+set :branch, 'update_nokogiri'
```

Comment:
Just temporarily to check the deploy again before committing the update to master.  In case there were other things I needed to change.